src/syntax/syntax-check.rkt

```diff
@@ -99,15 +99,22 @@
     (define cite (dict-ref prop-dict ':cite))
     (unless (list? (syntax-e cite))
       (error! cite "Invalid :cite ~a; must be a list" cite))
-    (when (list? (syntax-e cite))
-      (for ([citation (syntax-e cite)] #:unless (identifier? citation))
-        (error! citation "Invalid citation ~a; must be a variable name" citation))))
+    (for ([citation (syntax-e cite)] #:unless (identifier? citation))
+      (error! citation "Invalid citation ~a; must be a variable name" citation)))
```

Comment:
I don't think this is vaild, `error!` doesn't throw an error (just records it). But also I'm not sure why Herbie is checking the `:cite` property anyway, it doesn't use it.